src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -347,23 +429,36 @@ public NetworkPeerState State
             }
         }
 
-        public IPAddress RemoteSocketAddress { get; private set; }
+        /// <summary>IP address and port of the connected peer.</summary>
         public IPEndPoint RemoteSocketEndpoint { get; private set; }
+
+        /// <summary>IP address part of <see cref="RemoteSocketEndpoint"/>.</summary>
+        public IPAddress RemoteSocketAddress { get; private set; }
+
+        /// <summary>Port part of <see cref="RemoteSocketEndpoint"/>.</summary>
         public int RemoteSocketPort { get; private set; }
+
+        /// <summary><c>true</c> if the peer connected to the node, <c>false</c> if the node connected to the peer.</summary>
         public bool Inbound { get; private set; }
 
-        public bool ReuseBuffer { get; private set; }
+        /// <summary>List of node's modules attached to the peer to receive notifications about various events related to the peer.</summary>
         public NetworkPeerBehaviorsCollection Behaviors { get; private set; }
+
+        /// <summary>Information about the peer including its network address, protocol version, time of last contact.</summary>
         public NetworkAddress PeerAddress { get; private set; }
 
         /// <summary>Last time in UTC the node received something from this peer.</summary>
         public DateTime LastSeen { get; set; }
 
+        /// <summary>Difference between the local clock and the clock that peer claims, or <c>null</c> if this information has not been initialized yet.</summary>
         public TimeSpan? TimeOffset { get; private set; }
 
-        internal readonly NetworkPeerConnection connection;
+        /// <summary>Component representing the network connection to the peer that is responsible for sending and receiving messages.</summary>
+        internal readonly NetworkPeerConnection Connection;
 
+        /// <summary>Statistics about the number of bytes transferred from and to the peer.</summary>
         private PerformanceCounter counter;
+        /// <summary>Statistics about the number of bytes transferred from and to the peer.</summary>
```

Comment:
Space?

Reply:
i actually tend to group public properties with their underlaying fields, seems like a good binding
but not sure everyone likes that